app/assets/javascripts/discourse/controllers/notifications.js.es6

```diff
@@ -1,6 +1,7 @@
-export default Ember.ArrayController.extend({
+const NotificationsController = Ember.ArrayController.extend({
```

Comment:
Why bother making it a constant if you are not going to use it? I prefer the export directly in this case unless you need to reference the class somehow within this file.